package.json

```diff
@@ -52,12 +52,13 @@
     "p-lazy": "^1.0.0",
     "prettier": "^1.5.3",
     "recast": "git://github.com/kalcifer/recast.git#bug/allowbreak",
+    "resolve-cwd": "^2.0.0",
     "rx": "^4.1.0",
     "supports-color": "^4.2.0",
     "webpack": "^3.3.0",
     "webpack-addons": "1.1.2",
     "webpack-addons-ylvis": "0.0.34",
-    "yargs": "^8.0.2",
+    "yargs": "^6.5.0",
```

Comment:
Why the downgrade?

Reply:
See slack message. https://github.com/yargs/yargs/issues/925